src/schemas/bulkImport.json

```diff
@@ -28,29 +30,40 @@
           "npmOrgMFA",
           "npmPublicationMFA",
           "upgradePathDocs"
-        ]
+        ],
+        "examples": ["softwareDesignTraining"]
       },
       "project_id": {
-        "type": "integer"
+        "description": "The project id",
+        "type": "integer",
+        "minimum": 1,
+        "examples": [123]
       },
       "is_subscribed": {
+        "description": "Flag to indicate if the project is already subscribed",
```

Comment:
@UlisesGascon is this description correct?

Reply:
Probably something like "Flag to indicate if the project is already subscribed to the policy"?

Reply:
updated